tests/tests_basic.yml

```diff
@@ -4,6 +4,20 @@
   hosts: all
   vars:
     podman_run_as_user: root
+    __podman_registries_conf_file: 50-systemroles.conf
+    __podman_registries_conf_system: >-
+      /etc/containers/registries.conf.d/{{ __podman_registries_conf_file }}
+    __podman_registries_conf_user: >-
+      .config/containers/registries.conf.d/{{ __podman_registries_conf_file }
+    podman_registries_conf:
+      aliases:
+        tier1test: myregistry.domain/test/test:tier1
+        tier2test: myregistry.domain/test/test:tier2
+    __podman_containers_conf_file: 50-systemroles.conf
+    __podman_containers_conf_system: >-
+      /etc/containers/containers.conf.d/{{ __podman_containers_conf_file }}
+    __podman_containers_conf_user: >-
+      .config/containers/containers.conf.d/{{ __podman_containers_conf_file }
```

Comment:
you shouldn't need to redefine these variables in the test.  We are including the role with `public: true` so all of these should be exported and available later in the test.

Reply:
oh right, I didn't see that in the changeset ... it failed originally before my rebase when they weren't defined. Fixing